clean_project/test_data/scriptdb.sql

```diff
@@ -0,0 +1,12 @@
+createdb -h localhost -U postgres index_db
```

Comment:
it's not a sql file if you put also bash commands here. you should make all these commands as SQL instructions, and even better would be if you comment each of these explaining what's going one